tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -977,7 +989,13 @@ def macroAndSet(self, kernel, tPA, tPB) -> Module:
               offset = "v[\\vgprTmp+0]"
 
             # offset * stride
-            macro.add(VMulLOU32(dst=destLo,
+            if isSwizzled:
+              macro.add(VMulLOU32(dst=destLo,
+                src0="s[\\sgprStrideSW%s]" % tc, \
+                src1=offset, \
+                comment="SWZ: mul d%u lower"%i))
+            else:
+              macro.add(VMulLOU32(dst=destLo,
```

Comment:
For swizzling, replace " **Coord-I * StrideA0I** " with " **Coord-I * Swizzled-Stride** "